packages/ui-kit/src/components/TimeSeries/TimeSeries.tsx

```diff
@@ -289,21 +291,21 @@ export const TimeSeriesComponent = React.forwardRef<HTMLDivElement, TimeSeriesPr
     React.useEffect(() => {
       function handlePropsMismatch() {
         if (isStatic && !labels && !values) {
-          // console.error('InvalidPropsError: You must pass either `labels` and `values` or `query` props') we will set logs as a feature later
+          log.error('InvalidPropsError: You must pass either `labels` and `values` or `query` props')
           setPropsMismatch(true)
           return
         }
 
         if (isStatic && (!labels || !values)) {
-          // console.error('InvalidPropsError: When passing the data via props you must pass both `labels` and `values`') we will set logs as a feature later
+          log.error('InvalidPropsError: When passing the data via props you must pass both `labels` and `values`')
```

Comment:
Please also document these changes in the changeset, as it's a behavior change: we will now log in more cases when invalid props are passed.